create_tailscale_sysext.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+export ARCH="${ARCH-x86-64}"
+SCRIPTFOLDER="$(dirname "$(readlink -f "$0")")"
+
+if [ $# -lt 2 ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+  echo "Usage: $0 VERSION SYSEXTNAME"
+  echo "The script will download tailscale binaries and create a sysext squashfs image with the name SYSEXTNAME.raw in the current folder."
+  echo "A temporary directory named SYSEXTNAME in the current folder will be created and deleted again."
+  echo "All files in the sysext image will be owned by root."
+  echo "To use arm64 pass 'ARCH=arm64' as environment variable (current value is '${ARCH}')."
+  "${SCRIPTFOLDER}"/bake.sh --help
+  exit 1
+fi
+
+VERSION="$1"
+SYSEXTNAME="$2"
+
+if [ "${ARCH}" = "x86-64" ]; then
+  ARCH="amd64"
+fi
+
+tarball="$(curl -fsSL "https://pkgs.tailscale.com/stable/?v=${VERSION}&mode=json" | jq -r ".Tarballs.${ARCH}")"
+
+URL="https://pkgs.tailscale.com/stable/${tarball}"
+
+TMP_DIR="$(mktemp -d)"
+
+#trap 'rm -rf "${TMP_DIR}"' EXIT
```

Comment:
```suggestion
TPMDIR="${SYSEXTNAME}/tmp"
mkdir -p "${TMPDIR}"
```
And when we remove this before the `bake.sh` call we don't need a trap.